content/docs/hello-world.md

```diff
@@ -1,50 +1,48 @@
 
-Every chapter in this guide builds on the knowledge introduced in earlier chapters. **You can learn most of React by reading the “Main Concepts” guide chapters in the order they appear in the sidebar.** For example, [“Introducing JSX”](/docs/introducing-jsx.html) is the next chapter after this one.
+Κάθε κεφάλαιο αυτού του οδηγού στηρίζεται στις γνώσεις που έχουν παρουσιασθεί σε προηγούμενα κεφάλαια. **Μπορείτε να μάθετε το μεγαλύτερο μέρος του React διαβάζοντας τα κεφάλαια οδηγών “Βασικές έννοιες” με τη σειρά που εμφανίζονται στην πλαϊνή γραμμή**. Για παράδειγμα, [“Εισαγωγή στο JSX”](/docs/introducing-jsx.html) είναι το επόμενο κεφάλαιο μετά από αυτό.
 
-## Knowledge Level Assumptions {#knowledge-level-assumptions}
+## Υποθέσεις Επίπεδου Γνώσης {#knowledge-level-assumptions}
 
-React is a JavaScript library, and so we'll assume you have a basic understanding of the JavaScript language. **If you don't feel very confident, we recommend [going through a JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level** and enable you to follow along this guide without getting lost. It might take you between 30 minutes and an hour, but as a result you won't have to feel like you're learning both React and JavaScript at the same time.
+Το React είναι μια βιβλιοθήκη JavaScript και έτσι θα υποθέσουμε ότι έχετε μια βασική κατανόηση της γλώσσας JavaScript. **Εάν δεν αισθάνεστε βέβαιοι, συνιστούμε [να περάσετε από ένα JavaScript tutorial](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) για να ελέγξετε το επίπεδο γνώσης σας** και να σας επιτρέψουν να ακολουθήσετε αυτόν τον οδηγό χωρίς να χαθείτε. Μπορεί να σας πάρει μεταξύ 30 λεπτών και μιας ώρας, αλλά ως αποτέλεσμα δεν θα πρέπει να αισθάνεστε σαν να μαθαίνετε ταυτόχρονα τόσο το React όσο και το JavaScript.
 
->Note
+>Υπόδειξη
 >
->This guide occasionally uses some of the newer JavaScript syntax in the examples. If you haven't worked with JavaScript in the last few years, [these three points](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) should get you most of the way.
-
+>Αυτός ο οδηγός περιστασιακά χρησιμοποιεί ορισμένες από τις νεότερες συντάξεις JavaScript στα παραδείγματα. Εάν δεν έχετε δουλέψει με JavaScript τα τελευταία χρόνια, [αυτά τα τρία σημεία](https://gist.github.com/gaearon/683e676101005de0add59e8bb345340c) θα πρέπει να σας βοηθήσουν περισσότερο.
 
-## Let's Get Started! {#lets-get-started}
 
-Keep scrolling down, and you'll find the link to the [next chapter of this guide](/docs/introducing-jsx.html) right before the website footer.
+## Ας Ξεκινήσουμε! {#lets-get-started}
 
+Συνεχίστε να μετακινήστε προς τα κάτω και θα βρείτε τον σύνδεσμο στον [επόμενο κεφάλαιο αυτού του οδηγού](/docs/introducing-jsx.html) ακριβώς πριν το υποσέλιδο ιστότοπου.
```

Comment:
Thanks for the PR! 🎉
Ερώτηση 🤔"Γεια σου κόσμε" ή "Hello world"?
Στα παραδείγματα το έβαλα έτσι αλλά δεν το σκέφτηκα και πολύ εκείνη τη στιγμη...
Επίσης υποσέλιδο ή footer?

Reply:
Ουφ. Είμαι 50-50 και στις 2 περιπτώσεις.
Τα Ελληνικά εδώ μου φαίνονται εντάξει αν και το υποσέλιδο σπάνια το λέγαμε στο ΤΕΙ που πήγα αν βοηθάει.  Επίσης, το "Γεια σου κόσμε" εμφανίζεται συχνά στα βιβλία γλωσσών προγραμματισμού.

Στα Ισπανικά πήγαν με το δικό τους "Hello world". https://es.reactjs.org/docs/hello-world.html

Στην τελική, λέω να πάμε όσο περισσότερο γίνεται με τα Ελληνικά αν έχουν νόημα. Ωστόσο, δεν έχω θέμα αν το αφήσουμε στα Αγγλικά. 😄

Reply:
@john-d-pelingo  @gatzjames I think that the second one "Hello world" is much better.

Reply:
Κανονικά στην Ελληνική κοινότητα, το Hello World δεν είναι συνηθισμένο?

Ή μάλλον η σωστή έκφραση θα ήταν, "Γεια Σου, Κόσμο", επειδή είναι άμεσο αντικείμενο? 😄

Reply:
Ωραία, πάμε με "Hello world" στον τίτλο και στον κώδικα.

Το υποσέλιδο να φανταστώ γίνεται footer; Καμια αντίρρηση;

Reply:
+1 Για το "Ηello world" να μείνει στα Αγγλικά. Νομίζω πως αρκετοί που μαθαίνουν γλώσσες σε σχολές/σχολεία το μαθαίνουν έτσι.